Builder_/Components/Pizza.cs

```diff
@@ -0,0 +1,23 @@
+namespace builder_
+{
+    public class Pizza
```

Comment:
Good job declaring the `pizza` class but even though your solution is correct, it should have been two classes as written in the requirements.